substrate/frame/recovery/src/lib.rs

```diff
@@ -216,12 +206,28 @@ pub struct RecoveryConfig<BlockNumber, Balance, Friends> {
 	pub threshold: u16,
 }
 
-#[frame_support::pallet]
+/// The type of deposit
+#[derive(
+	Clone,
+	Eq,
+	PartialEq,
+	Encode,
+	Decode,
+	RuntimeDebug,
```

Comment:
We have a tracking issue to remove `RuntimeDebug`, Would suggest not adding it any further? https://github.com/paritytech/polkadot-sdk/pull/3107